packages/vscode-extension/src/extension.ts

```diff
@@ -814,13 +820,14 @@ async function runBackgroundAsyncTasks(
   await openWelcomePageAfterExtensionInstallation();
 
   await exp.initialize(context);
-  TreatmentVariableValue.previewTreeViewCommand = (await exp
-    .getExpService()
-    .getTreatmentVariableAsync(
-      TreatmentVariables.VSCodeConfig,
-      TreatmentVariables.PreviewTreeViewCommand,
-      true
-    )) as boolean | undefined;
+  TreatmentVariableValue.previewTreeViewCommand = true;
```

Comment:
Any reason to make it always true to have preview Tree view command?

Reply:
Thanks, fixed.